ext/handlers/filters/updates.go

```diff
@@ -0,0 +1,67 @@
+package filters
+
+import (
+	"github.com/PaulSonOfLars/gotgbot/v2"
+)
+
+type ChatMember func(u *gotgbot.ChatMemberUpdated) bool
+
+func (f ChatMember) And(f2 ChatMember) ChatMember {
```

Comment:
Looking for feedback, not changes for this PR; do you consider the And/Or/Not method to be necessary/helpful? Is it not easier to create a filter chain with a custom func instead? They seem confusing.

Reply:
imo it is good, however considering go's on the fly anonymous functions, it can be done that way too. But from the user's perspective, i think it is nice to have a "helper" thingy for it.

Reply:
Is it though? Or does it just add more confusion around how to pair filters? 

Personally, i think it would be much easier to read a filter using anonymous functions (which will use golang operators) rather than these, which have dubious operator precedence

Reply:
There can be users who understand this structural "And Or" filter implementation better than the maze of anonymous functions. If you consider that situation, they are good.

Reply:
But is that situation actually a real one?

Is it easier to read:
```go
func(u *gotgbot.ChatMemberUpdated) {
    return (filters.ChatMemberUserId(123) && filters.ChatMemberChatId(567)) || filters.ChatMemberChannel()
}
```
or
```go
filters.ChatMemberUserId(123).And(filters.ChatMemberChatId(567)).Or(filters.ChatMemberChannel())
```

At least the operator precendence of the first one is clear. The second one is dubious (i think it's the same? not sure)

Reply:
The thing is i m fine with either of them, its your library, you should be the one deciding what to keep.

Reply:
I'm asking you for an opinion, and a use case. If you don't have an opinion, all you need to do is say so.

Just because its "my library" (which is not how i consider open source work), that doesnt mean I always have the correct opinions. Which is why I ask other people.

Reply:
> But is that situation actually a real one?
> 
> Is it easier to read:
> 
> ```go
> func(u *gotgbot.ChatMemberUpdated) {
>     return (filters.ChatMemberUserId(123)(u) && filters.ChatMemberChatId(567)(u)) || filters.ChatMemberChannel()(u)
> }
> ```
> 
> or
> 
> ```go
> filters.ChatMemberUserId(123).And(filters.ChatMemberChatId(567)).Or(filters.ChatMemberChannel())
> ```
> 
> At least the operator precendence of the first one is clear. The second one is dubious (i think it's the same? not sure)

If I may, I think it's better to use golang operators, for both readability and ease of use. the "And Or" implementation is for limited languages like java.

Reply:
> If I may, I think it's better to use golang operators, for both readability and ease of use. the And/Or/Not methods are for limited languages like java.

Thanks for your input - I absolutely agree. I was looking for external opinions, so this is really appreciated. :)

I intend to refactor and remove them from all filters in a later PR, to keep things consistent